src/Operation/EstimatedDocumentCount.php

```diff
@@ -90,18 +115,53 @@ public function __construct($databaseName, $collectionName, array $options = [])
      */
     public function execute(Server $server)
     {
-        return $this->count->execute($server);
+        $command = $this->createCommand($server);
+        if ($command instanceof Aggregate) {
+            try {
+                $cursor = $command->execute($server);
+            } catch (CommandException $e) {
```

Comment:
I just noticed that ListIndexes uses `MongoDB\Driver\Exception\RuntimeException` for this logic. Reported that in [PHPLIB-636](https://jira.mongodb.org/browse/PHPLIB-636).